internal/handlers/common/distinct.go

```diff
@@ -32,9 +33,11 @@ type DistinctParams struct {
 	DB         string          `ferretdb:"$db"`
 	Collection string          `ferretdb:"collection"`
 	Key        string          `ferretdb:"key"`
-	Filter     *types.Document `ferretdb:"query,opt"`
+	Filter     *types.Document `ferretdb:"-"`
 	Comment    string          `ferretdb:"comment,opt"`
 
+	Query any `ferretdb:"query,opt"`
```

Comment:
Could you add a small comment about the difference between `Query` and `Filter` please?

Reply:
Actually looking at this implementation again, I have a question, is there any scenario where we want `query` to be used outside the scope of this function? Because having seemingly similar fields is confusing for using it. I'm wondering if we should have a struct that we only use to call `ExtractParams`, this may be more confusing than having two similar fields though...

Reply:
> Actually looking at this implementation again, I have a question, is there any scenario where we want query to be used outside the scope of this function?
That's a temporary solution as we are going to change how we process those parameters.

Reply:
Great! Thanks for clarification :hugs: